dist/scripts/rhinostyle.min.js

```diff
@@ -29585,7 +29585,8 @@ var Input = function (_React$Component) {
           required = _props.required,
           size = _props.size,
           type = _props.type,
-          validationMessage = _props.validationMessage;
+          validationMessage = _props.validationMessage,
+          dataCy = _props.dataCy;
```

Comment:
Like the other pr, does everyone prefer `dataCy` or `dataCypress`?

Reply:
Yeah, dataCypress is more readable, i'm changing it.